pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -313,6 +314,36 @@ func getContainerLogs(ctx context.Context,
 	return records
 }
 
+// getLogWithStacktracing search for the first stack trace line and offset it by logLinesOffset
+func getLogWithStacktracing(logArray []string) string {
+	var log []string
+	var found bool
+
+	for idx := range logArray {
+		line := logArray[idx]
+		// if it already found the stack we just want to keep add the log till the end
+		if found {
+			log = append(log, line)
+			continue
+		}
+		// tries to find the stack message at the given line and add it to the log
+		if found = stackTraceRegex.MatchString(line); found {
+			log = append(log, line)
+			continue
+		}
+		// if it didn't find the line yet, keep add it as offset limiting by the arbitrary
+		// offset limit value
+		log = append(log, line)
+		if len(log) > int(logLinesOffset) {
+			// to avoid memory leaks (https://github.com/golang/go/wiki/SliceTricks#delete-without-preserving-order)
+			log[0] = ""
```

Comment:
We don't store a pointer or struct in the array, so I don't think there will be a memory leak, but I could be wrong.

Reply:
It is one line of code, @tremes. One line of code. I prefer to be sure if the cost is one line of code that doesn't impact anything.